hostprocess/calico/calico.yml

```diff
@@ -0,0 +1,238 @@
+      "plugins": [
+        {
+          "windows_use_single_network": true,
+          "type": "calico",
+          "mode": "vxlan",
+          "nodename": "__KUBERNETES_NODE_NAME__",
+          "nodename_file_optional": true,
+          "log_file_path": "c:/cni.log",
+          "log_level": "debug",
+
+          "vxlan_mac_prefix": "0E-2A",
+          "vxlan_vni": 4096,
+          "mtu": __CNI_MTU__,
+          "policy": {
+            "type": "k8s"
+          },
+
+          "log_level": "info",
+
+          "capabilities": {"dns": true},
+          "DNS":  {
+            "Search":  [
+              "svc.cluster.local"
+            ]
+          },
+
+          "datastore_type": "kubernetes",
+
+          "kubernetes": {
+            "kubeconfig": "__KUBECONFIG_FILEPATH__"
+          },
+
+          "ipam": {
+            "type": "calico-ipam",
+            "subnet": "usePodCidr"
+          },
+
+          "policies":  [
+            {
+              "Name":  "EndpointPolicy",
+              "Value":  {
+                "Type":  "OutBoundNAT",
+                "ExceptionList":  [
+                  "__K8S_SERVICE_CIDR__"
+                ]
+              }
+            },
+            {
+              "Name":  "EndpointPolicy",
+              "Value":  {
+                "Type":  "SDNROUTE",
+                "DestinationPrefix":  "__K8S_SERVICE_CIDR__",
+                "NeedEncap":  true
+              }
+            }
+          ]
+        }
+      ]
+
+    }
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: calico-node-windows
+  labels:
+    tier: node
+    app: calico
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: calico
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: calico
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: kubernetes.io/os
+                    operator: In
+                    values:
+                      - windows
+                  - key: kubernetes.io/arch
+                    operator: In
+                    values:
+                      - amd64
+      securityContext:
+        windowsOptions:
+          hostProcess: true
+          runAsUserName: "NT AUTHORITY\\system"
+      hostNetwork: true
+      serviceAccountName: calico-node
+      tolerations:
+      - operator: Exists
+        effect: NoSchedule
+        # Mark the pod as a critical add-on for rescheduling.
+      - key: CriticalAddonsOnly
+        operator: Exists
+      - effect: NoExecute
+        operator: Exists
+      initContainers:
+        # This container installs the CNI binaries
+        # and CNI network config file on each node.
+        - name: install-cni
+          image: sigwindowstools/calico-install:v3.20.0-hostprocess
+          args: ["$env:CONTAINER_SANDBOX_MOUNT_POINT/calico/install.ps1"]
```

Comment:
what interprets this envvar? normally, envvars would be specified in args as `$(CONTAINER_SANDBOX_MOUNT_POINT)/calico/install.ps1` (https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/#using-environment-variables-inside-of-your-config)